tests/coverage/Approaches.md

```diff
@@ -0,0 +1,153 @@
+# What coverage approaches are needed?
```

Comment:
We should have this on the website documentation as well. @mravanelli, @Gastron, @pplantinga, @Adel-Moumen  what do you think?

Reply:
Certainly it can be migrated from here to the website.